[SRE-295] Update the readme to indicate that a legacy module (#5)
hoverinc/terraform-google-vault: README.md

# Vault on GCE Terraform Module

> **IMPORTANT:** this is a legacy module that will be deprecated once their dependencies are gone

Reviewer: ```suggestion
**IMPORTANT:** this is a legacy module that will be deprecated once it is no longer depended on.
```

Author: Nothing better that a review of a native speaker! :-)

Reviewer: I am constantly impressed by Neidson's, and Andres', and your english. Happy to help.
